docs/source/guides/speed.rst

```diff
@@ -76,8 +76,8 @@ Prefer DDP over DP
 Whereas :class:`~pytorch_lightning.plugins.training_type.DDPPlugin` only performs 1 transfer to sync gradients, making DDP MUCH faster than DP.
 
 
-When using DDP set find_unused_parameters=False
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When using DDP or DDPSpawn, set find_unused_parameters=False
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Shouldn't this section also mention the availability of the `str`? Or do we mention it somewhere else?

Reply:
`PluginsRegistry` will have a separate section with multiple DeepSpeed aliases and this.